index.bs

```diff
@@ -89,30 +112,13 @@ interface VRDisplay : EventTarget {
    */
   readonly attribute DOMString displayName;
 
-  /**
-   * Populates the passed VRFrameData with the information required to render
-   * the current frame.
-   */
-  boolean getFrameData(VRFrameData frameData);
+  VRAttachedFrameOfReference createAttachedFrameOfReference();
```

Comment:
Maybe the term "frame of reference" could also be explained in the terminology list above. I know, it does not solve the naming issue but I think it might be useful for developers (if we agree upon the name "frame of reference" and its need).